lib/configs/recommended.js

```diff
@@ -104,7 +105,15 @@ module.exports = {
     'space-in-parens': ['error', 'never'],
     'space-infix-ops': ['error', { 'int32Hint': true }],
     'space-unary-ops': 'error',
-    'strict': ['error', 'global']
+    'strict': ['error', 'global'],
+    'disallow-methods/disallow-methods': [
+      'warn', [
```

Comment:
I missed this in your other PR -- why warn instead of error?

Reply:
Changing to error per offline slack convo.